governance.md

```diff
@@ -1,149 +1,285 @@
+    - responsive to [code review](contributors/devel/collab.md) requests as per
+      [community expectations](contributors/devel/community-expectations.md)
+    - mentor REVIEWERS and CONTRIBUTORS
+  - Benefits
+    - TBD: swag
+- **MAINTAINER**:
+  - includes all requirements, expectations, and benefits of APPROVERS
+  - Requirements
+    - APPROVER for some part of the codebase for at least 3 months
+    - on project for at least 1 year
+    - authored and/or reviewed at least 50 merged non-trivial PRs, including at least 20 as the primary
+      reviewer
+    - familiar enough with the codebase and project conventions to be in the repo’s top-level OWNERS
+      file `maintainers` list
+    - committed to the project: Kubernetes is a very high-volume project, with hundreds of PRs and issues
+      per week, so expect a significant time commitment 
+    - nominated by a Champion (see below) from the existing MAINTAINERS, who will find a
+      Sponsor (see below) from LEADS, and (assuming the person accepts) will email the existing
+      MAINTAINERS via their mailing list (e.g., `kubernetes-maintainers@googlegroups.com`), with
+      a summary of contributions to the project, current project responsibilities, and links to
+      queries for merged and assigned PRs
+    - at least 3 of the MAINTAINERS must approve the application, with no objections
+    - the application expires after 2 weeks if not enough approvals are granted
+  - Expectations
+    - review PRs
+    - fix bugs
+    - maintain and improve health and quality of the project
+    - provide user support
+    - fulfill other project responsibilities and roles
+    - mentor APPROVERS, REVIEWERS, and CONTRIBUTORS
+  - Benefits
+    - added to the repo’s maintainers github team (e.g., [`kubernetes-maintainers`](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers))
+    - write access to the repo: assign issues/PRs, add/remove labels and milestones, 
+      edit issues and PRs, edit wiki, create/delete labels and milestones
+    - technically can approve any PR and cause it to be merged by the submit queue, but expected to
+      respect OWNERS files
+- **OWNER**:
+  - includes all requirements, expectations, and benefits of MAINTAINERS
+  - Requirements
+    - PR and tenure requirements and nomination process TBD
+    - familiar enough with some area/component to be in its top-level
+      [OWNERS](contributors/devel/owners.md) file `owners` list
+    - may be asked to become an area/component OWNER as a precondition for accepting the 
+      contribution of a new component or other major function
+    - TBD: which areas/components need OWNERS? Examples: Apiserver, Kubelet, Kubectl
+  - Expectations
+    - mentor APPROVERS, REVIEWERS, and CONTRIBUTORS
+  - Benefits
+    - design/proposal approval authority for some area of the project, though escalation to LEADS
+      is still possible, and beta/GA APIs must still be vetted by API REVIEWERS and APPROVERS
+- **LEAD**:
```

Comment:
It does de facto exist. 

Originally I had it as the top-level approvers, but ran into a situation in less than a week after writing that where conflating the two caused a problem.

The google3 OWNERS mechanism has at least a convention for adding top-level OWNERS for approving cross-cutting code changes, such as changes to the build system, automated refactorings, etc.